knexfile.js

```diff
@@ -8,10 +8,10 @@ module.exports = {
   development: {
     client: 'pg',
     connection: {
+      port: 5432,
       database: 'pet_furriendly_db',
       user:     'postgres',
-      password: 'Turing2301',
-      port: 5432
+      password: 'Turing2301'
```

Comment:
For this project, it's okay to have the password showing up, but when we are working with real users, you might want to hide this password in the` .env` file.